apps/web/src/app/components/file-password-prompt.component.html

```diff
@@ -0,0 +1,58 @@
+<div class="modal fade" role="dialog" aria-modal="true" aria-labelledby="confirmVaultImport | i18n">
```

Comment:
This expression needs to be evaluated - at the moment it's just a string. e.g.
```suggestion
<div class="modal fade" role="dialog" aria-modal="true" [attr.aria-labelledby]="confirmVaultImport | i18n">
```

Note that you need to prefix bound aria attributes with `attr.`, this is a weird Angular thing. [reference](https://angular.io/guide/accessibility#accessibility-attributes)